### ISSUE TITLE
Look at media options as smaller or bigger than

### ISSUE THREAD

**Issue by ryssbowh** [mike-diamond/media-query-splitting-plugin#19]
Hi,

I've been trying to use this package with external libraries (such as bootstrap) and realised that the numbers set in the config must match exactly in order to be dispatched to different css files.
So I tried modifying your package so it dispatch queries if the numbers are smaller or bigger than the ones set in config. In other words, make it work even with media queries that don't match exactly the ones set in config.

I tried it with bootstrap, seems to work, even though I've not test it thoroughly yet.

If you're interested with this idea, I can do a pull request, the commit is [there](https://github.com/ryssbowh/media-query-splitting-plugin/commit/7767e33fd28e3b1b27610b23c331de0ad2679126).

Cheers

**Comment by mike-diamond** [comment]
Yes, thank you, this feature will be implemented

**Comment by ryssbowh** [comment]
Doing a pull request now, here some testing I've made with the following config :

mobileEnd: 568,
tabletPortraitEnd: 768,
tabletLandscapeEnd: 1024,

source :
```
body {
	background: 'main';
}
@media all and (max-width: 569px) {
	body{background: 'mobile';}
}
@media all and (max-width: 768px) {
	body{background: 'mobile portrait';}
}
@media all and (max-width: 1024px) {
	body{background: 'mobile portrait landscape';}
}
@media all and (min-width: 569px) {
	body{background: 'landscape portait desktop';}
}
@media all and (min-width: 1024px) {
	body{background: 'landscape desktop';}
}
@media all and (min-width: 569px) and (max-width: 1024px) {
	body{background: 'portrait landscape';}
}
@media all and (min-width: 300px) {
	body{background: 'mobile portrait landscape desktop';}
}
@media all and (min-width: 600px) {
	body{background: 'portrait landscape desktop';}
}
@media all and (min-width: 800px) {
	body{background: 'landscape desktop';}
}
@media all and (min-width: 1200px) {
	body{background: 'desktop';}
}
@media all and (max-width: 600px) {
	body{background: 'mobile portrait';}
}
@media all and (max-width: 800px) {
	body{background: 'mobile portrait landscape';}
}
@media all and (max-width: 1200px) {
	body{background: 'mobile portrait landscape desktop';}
}
@media all and (min-width:400px) and (max-width: 600px) {
	body{background: 'mobile portrait';}
}
@media all and (min-width:600px) and (max-width: 800px) {
	body{background: 'portrait landscape';}
}
@media all and (min-width:800px) and (max-width: 1200px) {
	body{background: 'landscape desktop';}
}
@media all and (min-width:1100px) and (max-width: 1300px) {
	body{background: 'desktop';}
}
```
mobile.css
```
@media all and (max-width: 569px) {
  body {
    background: 'mobile';
  }
}

@media all and (max-width: 768px) {
  body {
    background: 'mobile portrait';
  }
}

@media all and (max-width: 1024px) {
  body {
    background: 'mobile portrait landscape';
  }
}

@media all and (min-width: 300px) {
  body {
    background: 'mobile portrait landscape desktop';
  }
}

@media all and (max-width: 600px) {
  body {
    background: 'mobile portrait';
  }
}

@media all and (max-width: 800px) {
  body {
    background: 'mobile portrait landscape';
  }
}

@media all and (max-width: 1200px) {
  body {
    background: 'mobile portrait landscape desktop';
  }
}

@media all and (min-width: 400px) and (max-width: 600px) {
  body {
    background: 'mobile portrait';
  }
}
```
tabletPortrait.css
```
@media all and (max-width: 768px) {
  body {
    background: 'mobile portrait';
  }
}

@media all and (max-width: 1024px) {
  body {
    background: 'mobile portrait landscape';
  }
}

@media all and (min-width: 569px) {
  body {
    background: 'landscape portait desktop';
  }
}

@media all and (min-width: 569px) and (max-width: 1024px) {
  body {
    background: 'portrait landscape';
  }
}

@media all and (min-width: 300px) {
  body {
    background: 'mobile portrait landscape desktop';
  }
}

@media all and (min-width: 600px) {
  body {
    background: 'portrait landscape desktop';
  }
}

@media all and (max-width: 800px) {
  body {
    background: 'mobile portrait landscape';
  }
}

@media all and (max-width: 1200px) {
  body {
    background: 'mobile portrait landscape desktop';
  }
}

@media all and (min-width: 600px) and (max-width: 800px) {
  body {
    background: 'portrait landscape';
  }
}
```
tabletLandscape
```
@media all and (max-width: 1024px) {
  body {
    background: 'mobile portrait landscape';
  }
}

@media all and (min-width: 569px) {
  body {
    background: 'landscape portait desktop';
  }
}

@media all and (min-width: 1024px) {
  body {
    background: 'landscape desktop';
  }
}

@media all and (min-width: 569px) and (max-width: 1024px) {
  body {
    background: 'portrait landscape';
  }
}

@media all and (min-width: 300px) {
  body {
    background: 'mobile portrait landscape desktop';
  }
}

@media all and (min-width: 600px) {
  body {
    background: 'portrait landscape desktop';
  }
}

@media all and (min-width: 800px) {
  body {
    background: 'landscape desktop';
  }
}

@media all and (max-width: 1200px) {
  body {
    background: 'mobile portrait landscape desktop';
  }
}

@media all and (min-width: 800px) and (max-width: 1200px) {
  body {
    background: 'landscape desktop';
  }
}
```
desktop.css
```
@media all and (min-width: 569px) {
  body {
    background: 'landscape portait desktop';
  }
}

@media all and (min-width: 1024px) {
  body {
    background: 'landscape desktop';
  }
}

@media all and (min-width: 300px) {
  body {
    background: 'mobile portrait landscape desktop';
  }
}

@media all and (min-width: 600px) {
  body {
    background: 'portrait landscape desktop';
  }
}

@media all and (min-width: 800px) {
  body {
    background: 'landscape desktop';
  }
}

@media all and (min-width: 1200px) {
  body {
    background: 'desktop';
  }
}

@media all and (max-width: 1200px) {
  body {
    background: 'mobile portrait landscape desktop';
  }
}

@media all and (min-width: 800px) and (max-width: 1200px) {
  body {
    background: 'landscape desktop';
  }
}

@media all and (min-width: 1100px) and (max-width: 1300px) {
  body {
    background: 'desktop';
  }
}
```

**Comment by mike-diamond** [comment]
Implemented in v 2.0.0
